### ISSUE TITLE
Breakdown view: Character casting assignments disappear after navigation/reload

### ISSUE THREAD

**Issue by herrschmidt** [cgwire/kitsu#1606]
**Context**

Studio name: Raumkapsel
Kitsu version: 0.20.0
Production type: Short
Deployment: Testing Kitsu with Docker Compose using registry.gitlab.com/mathbou/docker-cgwire images (latest tag)

**Describe the bug**
Character casting assignments in the breakdown view are not displaying correctly despite being properly stored in the database. Specifically:

1. When assigning characters to shots, the assignments are successfully saved to the database (confirmed via SQL query)
2. However, the character assignments are not displayed in the UI, even after page refresh
3. Using "Export current view" shows no character assignments
4. Using "Export" (direct database export) correctly shows all character assignments

This indicates that while the data is properly stored in the database, there's an issue with how the frontend retrieves and displays the casting data.

**Database Evidence**
The following SQL query confirms the data is correctly stored:
```sql
SELECT el.*, e_in.name as shot_name, e_out.name as character_name
FROM entity_link el
JOIN entity e_in ON el.entity_in_id = e_in.id
JOIN entity e_out ON el.entity_out_id = e_out.id
WHERE el.created_at > '2024-12-13 15:15:00'
ORDER BY el.created_at DESC;
```

Results show proper linking:
- SH01 linked to "Elephant"
- SH02 linked to "Duck"
- SH03 linked to both "Duck" and "Elephant"

All shots belong to the same sequence (SQ01) and have proper `label = 'animate'` and `nb_occurences = 1`.

**Steps to Reproduce**
1. Go to the breakdown view
2. Assign a character to a shot (e.g., "Duck" to "SH02")
3. The assignment appears to work initially
4. Navigate away or apply a filter
5. Return to the breakdown view
6. Observe that no characters are displayed

**Expected behavior**
Character assignments should persist and be visible in the UI, matching what is stored in the database.

**Desktop:**
- OS: Windows
- Browser: Firefox 133.0.3 (64-bit) and Chrome Version 131.0.6778.140 (Official Build) (64-Bit)

**Additional context**
- The issue appears to be related to how the frontend retrieves casting data via the `/data/projects/{project_id}/sequences/{sequence_id}/casting` endpoint
- The data is correctly stored in the database but not properly retrieved/displayed
- This affects both normal navigation and page refreshes
- The database export feature correctly shows all assignments, confirming the data exists in the database

**Comment by NicoPennec** [comment]
Thank you @herrschmidt for the full report 🙏 

The issue is fixed on Kitsu v0.20.1

**Comment by herrschmidt** [comment]
> Thank you @herrschmidt for the full report 🙏
> 
> The issue is fixed on Kitsu v0.20.1

Wow, @NicoPennec , that was fast! Thank you very much!!
